testSetup.js

```diff
@@ -1 +0,0 @@
-require('react-native-mock/mock');
```

Comment:
This seems unnecessary since the `react-native` preset for jest already mocks React Native